charon/src/bin/charon-driver/driver.rs

```diff
@@ -131,6 +134,9 @@ impl Callbacks for CharonCallbacks {
                 }
             }
         });
+
+        // Mutate other fields in Config
+        MutateConfig::new().do_operation(config);
```

Comment:
This is more complicated than needed. Why not just call `disabled_mir_passes(config);` etc here?

Reply:
Done. I've also cleaned up the history commits with MutateConfig pruned.